Sources/Public/DotLottie/DotLottieFile.swift

```diff
@@ -25,14 +25,61 @@ public final class DotLottieFile {
     try decompress(data: data, to: fileUrl)
   }
 
-  // MARK: Internal
+  // MARK: Public
 
   /// Definition for a single animation within a `DotLottieFile`
-  struct Animation {
-    let animation: LottieAnimation
-    let configuration: DotLottieConfiguration
+  public struct Animation {
+    public let animation: LottieAnimation
+    public let configuration: DotLottieConfiguration
+  }
+
+  /// The `LottieAnimation` and `DotLottieConfiguration` for the given animation ID in this file
+  public func animation(for id: String? = nil) -> DotLottieFile.Animation? {
```

Comment:
In the `public` function, could we change `for id: String?` to `for id: String`? 

If we need a helper for handling `nil` IDs, can we keep that private?